docs/guide/preferences.md

```diff
@@ -29,7 +29,8 @@ These user folders should contain projects resources that can be used in the cur
 Multiple user folders can be added by separating them with `:` (or `;` on Windows).
 They may contain multiple sub-folders, each one associated to one sub-project (which should respect the [Standard File Hierarchy of a Project](the-standard-file-hierarchy-of-a-project.md)).
 Alternatively, the environment variable `WEBOTS_EXTRA_PROJECT_PATH` can be used.
-If the `Extra project path` parameter and the `WEBOTS_EXTRA_PROJECT_PATH` environment variable are set, then both will be considered.
+If the `Extra project path` parameter and the `WEBOTS_EXTRA_PROJECT_PATH` environment variable are set, then both will be considered, having extra priority the paths set in the `Preferences` setting.
```

Comment:
The second part of this sentence sounds odd to me, may I suggest this formulation?
```suggestion
If the `Extra project path` parameter and the `WEBOTS_EXTRA_PROJECT_PATH` environment variable are set, then both will be considered, with the priority given to the paths set in the `Preferences` setting.
```

Reply:
Sounds good, I applied your suggestion.